src/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests/Services/FrontEndServiceTests.cs

```diff
@@ -18,8 +18,44 @@ public class FrontEndServiceTests
 {
     private const string HttpClientName = "PublicApp";
 
+    private const string TableJson = @"
```

Comment:
Nice!

Just wondering if we were to do this again using larger JSON examples, would it be preferable to have the JSON as an actual `.json` file within the `Resources` folder and grabbing it up with ` var jsonPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!,"Resources" + Path.DirectorySeparatorChar + filename);` and reading in the file contents from there?  We do similar in `ProcessorStage3Tests.cs`, for example.

Reply:
I was already considering doing it so I've made this change now! It also means the same table test json isn't being duplicated across two test classes 🙂 .